creator-node/src/routes/tracks.js

```diff
@@ -959,6 +959,69 @@ router.get(
  * Gets a streamable mp3 link for a track by encodedId. Supports range request headers.
  * @dev - Wrapper around getCID, which retrieves track given its CID.
  **/
-router.get('/tracks/cidstream/:CID', contentAccessMiddleware, getCID)
+router.get(
+  '/tracks/cidstream/:CID',
+  contentAccessMiddleware,
+  async (req, res, next) => {
+    const libs = req.app.get('audiusLibs')
+    const redisClient = req.app.get('redisClient')
+    const delegateOwnerWallet = config.get('delegateOwnerWallet')
+
+    const trackId = req.query.trackId
+    const CID = req.params.CID
+
+    if (!trackId) {
+      return sendResponse(
+        req,
+        res,
+        errorResponseBadRequest('Please provide a track ID')
+      )
+    }
+
+    const trackIdBlacklisted = await BlacklistManager.trackIdIsInBlacklist(
```

Comment:
can you add a comment about why we check both trackID and CID against blacklist

Reply:
chatted with @jonaylor89 and he will handle this later. need to merge this PR for now

Reply:
cool